packages/sol6/src/yang/etsi-nfv-vnf.yang

```diff
@@ -1373,24 +1373,6 @@ submodule etsi-nfv-vnf {
                 "GS NFV IFA011: Section 7.1.8.12, AffinityOrAntiAffinity
                  information element.";
             }
-            leaf type {
-              type affinity-type;
-              description
-                "Specifies whether the rule is an affinity rule or an
-                 anti-affinity rule.";
-              reference
-                "GS NFV IFA011: Section 7.1.8.12, AffinityOrAntiAffinity
-                 information element.";
-            }
-            leaf scope {
-              type affinity-scope;
-              description
-                "Specifies the scope of the rule, possible values are
-                 'NFVI-PoP', 'Zone', 'ZoneGroup', 'NFVI-node'";
-              reference
-                "GS NFV IFA011: Section 7.1.8.12, AffinityOrAntiAffinity
-                 information element.";
-            }
```

Comment:
IFA011 defines type and scope as enums, not pointers. Why do you think they should be pointers?

groupId M 1 Identifier 
type       M 1 Enum 
scope    M 1 Enum

Reply:
Yes, in the vnf-profile, it should just be a pointer to the group that is in the df (deployment-flavor).
See 7.1.8.3 VduProfile information element in the IFA11, it should only contain a identifier to a affinityOrAntiAffinityGroupId, not an affinityOrAntiAffinityGroup element